webapi/src/main/scala/org/knora/webapi/responders/v2/SearchResponderV2.scala

```diff
@@ -870,6 +884,36 @@ final case class SearchResponderV2Live(
           .as(countResponse.results.bindings.head.rowMap("count"))
 
     } yield ResourceCountV2(count.toInt)
+
+  private def ensureResourceClassIri(resourceClassIri: SmartIri): Task[SmartIri] = {
+    val errMsg = s"Resource class IRI <$resourceClassIri> is not a valid Knora API v2 entity IRI"
+    if (resourceClassIri.isKnoraApiV2EntityIri) {
+      iriConverter.asInternalSmartIri(resourceClassIri).orElseFail(BadRequestException(errMsg))
+    } else { ZIO.fail(BadRequestException(errMsg)) }
+  }
+
+  private def ensureStandoffClass(standoffClassIri: SmartIri): Task[SmartIri] = {
```

Comment:
```suggestion
  private def ensureStandoffClassIri(standoffClassIri: SmartIri): Task[SmartIri] = {
```

Reply:
I appreciate the effort in renaming. However, I will not integrate it using GitHub as I do not have compiler support and in this case you have missed renaming the caller side.